File: s22557858.pl
#!/usr/bin/perl
use strict;
use warnings;

# Check if filename and word are provided
if (@ARGV != 2) {
    print "Usage: perl script.pl filename word\n";
    exit 1;
}

my ($filename, $word) = @ARGV;

# Read the file
open my $fh, '<', $filename or die "Could not open '$filename' $!\n";
my $content = do { local $/; <$fh> };

# Convert to lowercase and remove non-alphabetic characters
$content = lc $content;
$content =~ s/[^a-z]//g;

# Count occurrences
my $count = () = reverse($content) =~ /(?=reverse($word))/g;

# Output
print "Matches: $count\n";

File: s22557858.php
<?php
// Check if filename and word are provided
if ($argc != 3) {
    echo "Usage: php script.php filename word\n";
    exit(1);
}

$filename = $argv[1];
$word = $argv[2];

// Read the file
$content = file_get_contents($filename);

// Convert to lowercase and remove non-alphabetic characters
$content = strtolower($content);
$content = preg_replace("/[^a-z]/", "", $content);

// Count occurrences
$count = substr_count(strrev($content), strrev($word));

// Output as HTML
echo "<html><body>Matches: $count</body></html>\n";
?>